src/Tools/RepoUtil/RepoUtil.csproj

```diff
@@ -1,19 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\..\build\Targets\SettingsSdk.props" />
   <PropertyGroup>
     <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <ProjectGuid>{1CA184D3-89CB-4074-BEC5-F8AEBA657D41}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>RepoUtil</RootNamespace>
-    <AssemblyName>RepoUtil</AssemblyName>
     <TargetFramework>net46</TargetFramework>
     <RuntimeIdentifiers>win7</RuntimeIdentifiers>
     <UseVSHostingProcess>false</UseVSHostingProcess>
-    <TargetFrameworkProfile />
     <PublishUrl>publish\</PublishUrl>
```

Comment:
* These properties: https://github.com/srivatsn/roslyn/blob/551b228610b9463502b9cf69fd75289c874cbaaf/src/Tools/RepoUtil/RepoUtil.csproj#L12-L26
* This item group: https://github.com/srivatsn/roslyn/blob/551b228610b9463502b9cf69fd75289c874cbaaf/src/Tools/RepoUtil/RepoUtil.csproj#L46-L57

Cleaning them up is probably a separate issue though, unrelated to the transition to CPS.